Remotion/ObjectBinding/Web/Res/Themes/NovaViso/Image/sprite.svg

```diff
@@ -92,4 +92,15 @@
       <use href="#SingleChevron" />
     </g>
   </svg>
```

Comment:
I've noticed that the checkbox uses transparent instead of white background. we need to update this.